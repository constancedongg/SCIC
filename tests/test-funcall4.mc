void func foo(int x, int y, bool a, bool b) {
    if (a && b) 
        print(x + y);
    else 
        print(x - y);
}

int func main() {
    foo(1, 2, true, false);   /* -1 */
    foo(1, 2, true, true);    /*  3 */ 
    return 0;
}