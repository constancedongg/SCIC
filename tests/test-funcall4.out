0.1
1.09
1.09
0.2
1.19
0.2